java/component/keyple-core/src/main/java/org/eclipse/keyple/core/seproxy/plugin/local/AbstractObservableLocalReader.java

```diff
@@ -182,8 +318,9 @@ public boolean isSePresent() throws KeypleIOReaderException {
      *        call to startSeDetection is made.
      */
     public void startSeDetection(ObservableReader.PollingMode pollingMode) {
```

Comment:
the startSeDetection(ObservableReader.PollingMode pollingMode) method should be defined as final